src/backend/vulkan/src/lib.rs

```diff
@@ -395,21 +401,18 @@ impl hal::PhysicalDevice<Backend> for PhysicalDevice {
                 self.instance
                     .0
                     .create_device(self.handle, &info, None)
-                    .map_err(|err| match err {
-                        ash::DeviceError::LoadError(err) => panic!("{:?}", err),
-                        ash::DeviceError::VkError(err) => Into::<result::Error>::into(err),
-                    })
+                    .map_err(|err| Into::<result::Error>::into(err))
```

Comment:
nit: could just be `map_err(Into::<result::Error>::into)`

Reply:
Fixed